deps/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/deps/util/ToolsTest.java

```diff
@@ -63,464 +54,4 @@ public void isNullOrEmptyInputNotEmpty()
         // Assert
         assertEquals(expResult, result);
     }
-
-    // Tests_SRS_SDK_JAVA_TOOLS_12_004: [The function shall return true if the input is null]
-    @Test
```

Comment:
Again, these tests covered code that wasn't being used anywhere, so they are safe to delete